SharedPackages/AIChat/Sources/AIChat/iOS/Views/QuickActions/AIChatQuickActionChipView.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  AIChatQuickActionChipView.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if os(iOS)
+import DesignResourcesKit
+import UIKit
+
+// MARK: - View
+
+/// A pill-shaped chip view displaying a quick action with icon and label.
+public final class AIChatQuickActionChipView: UIView {
+
+    // MARK: - Constants
+
+    private enum Constants {
+        static let height: CGFloat = 36
+        static let cornerRadius: CGFloat = 8
+        static let iconLeadingPadding: CGFloat = 8
+        static let iconTopPadding: CGFloat = 10
+        static let trailingPadding: CGFloat = 12
+        static let iconSize: CGFloat = 16
+        static let iconLabelSpacing: CGFloat = 6
+        static let borderWidth: CGFloat = 1
+        static let highlightAlpha: CGFloat = 0.1
+    }
+
+    // MARK: - Properties
+
+    var onTap: (() -> Void)?
+
+    // MARK: - UI Components
+
+    private lazy var iconView: UIImageView = {
```

Comment:
Just wanted to say that I always like when people create lazy properties for UI elements. :)